src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionxpathfilter/InputSuppressionXpathFilterEscape.java

```diff
@@ -0,0 +1,35 @@
+package com.puppycrawl.tools.checkstyle.filters.suppressionxpathfilter;
+
+public class InputSuppressionXpathFilterEscape {
+
+    String quoteChar = "\"escaped\"" +
+            "2"; // violation
+
+    String lessChar = "<escaped" +
+            "3"; // violation
+
+    String ampersandChar = "&escaped" +
+            "4"; // violation
+
+    String greaterChar = ">escaped" +
+            "5"; // violation
+
+    String newLineChar = "escaped\n" +
+            "6"; // violation
+
+    String specialChar = "escaped\r" +
+            "6"; // violation
+
+    String aposChar = "'escaped'" +
+            "7"; // violation
+
+    char a = '&'; // warning
+
+    char b = '\"'; // warning
+
+    char c = '\''; // warning
+
+    char d = '<'; // warning
+
+    char e = '>'; // warning
```

Comment:
`// warning` -> // violation

Please move char examples to other input file

Reply:
Ok, On it.

Reply:
Sir, can you please tell how to solve `StyleValidationCommentInInputs` error on this file
```
package com.puppycrawl.tools.checkstyle.filters.suppressionxpathfilter;

/*
 * Config: Default
 */
public class InputSuppressionXpathFilterEscapeChar {

    char a = '&'; // violation

    char b = '\"'; // violation

    char c = '\''; // violation

    char d = '<'; // violation

    char e = '>'; // violation
}
```

Reply:
I am struggling to solve this error for more than 2 hrs, but unable to pass this check. Also, looked at regexp of this check, it matches (according to me) to the [checkstyle_input_checks.xml](https://github.com/checkstyle/checkstyle/blob/7c7958a6004f3a0bd4c9444027ecdffc8ae9353f/config/checkstyle_input_checks.xml#L29) but still gives this error.

Reply:
it is weird. Please commit your changes and push them, I can checkout and take a look

Reply:
I had pushed changes Sir. Please take a look.

Reply:
After commiting the issue seems to be solved automatically. Sorry for bothering you Sir.

Reply:
Changes done.